srcpkgs/NetworkManager-l2tp/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'NetworkManager-l2tp'
+pkgname=NetworkManager-l2tp
+version=1.8.2
+revision=1
+build_style=gnu-configure
+configure_args="--runstatedir=/run"
+hostmakedepends="pkg-config intltool glib-devel"
```

Comment:
Shouldn't `glib-devel` be in `makedepends`, from a very quick scan of the Makefile.am?

Reply:
If it needs `glib-compile-resources`, it needs `glib-devel` on host. I saw what looked like a call to it.

Reply:
According to https://github.com/void-linux/void-packages/blob/master/srcpkgs/libglib-devel, `libglib-devel` is a symlink to `glib-devel`. So I'm guessing `glib-devel` doesn't need to be specified if `libglib-devel` is used. But happy to change it if required.

I used a combination of  `makedepends` from NetworkManager-pptp and NetworkManager-openvpn, plus `libressl-devel` and `nss-devel`.

Reply:
@ericonr that's kinda silly IMO, that the binaries are in there, but w/e, you're right

@dkosovic So `glib-devel` is still good here. But, for future reference `glib-devel` actually contains different content to `libglib-devel`. This is because, in void's packaging system, a symlink just means "use this template". As you can see [here](https://github.com/void-linux/void-packages/blob/master/srcpkgs/glib/template#L26) and [here](https://github.com/void-linux/void-packages/blob/master/srcpkgs/glib/template#L38), they are both from the same template `glib`. It's just that the files get moved out to those "subpackages". So, despite them being symlinks, they actually have different content. Two subpackages from the same template can be necessary, as they are in this case :) HTH